cmd/sovereignnode/chainSimulator/sovereignChainSimulator.go

```diff
@@ -23,12 +23,29 @@ import (
 
 // ArgsSovereignChainSimulator holds the arguments for sovereign chain simulator
 type ArgsSovereignChainSimulator struct {
-	SovereignExtraConfig config.SovereignConfig
-	ChainSimulatorArgs   chainSimulator.ArgsChainSimulator
+	SovereignConfigPath string
+	ChainSimulatorArgs  *chainSimulator.ArgsChainSimulator
```

Comment:
I meant without internal extra struct:
```suggestion
	 *chainSimulator.ArgsChainSimulator
```